controllers/stocks.js

```diff
@@ -37,8 +37,28 @@ const fetchStocks = async (req, res) => {
     return res.boom.badImplementation('An internal server error occurred')
   }
 }
+/**
+ * Fetches all the stocks of the user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const getSelfStocks = async (req, res) => {
+  try {
+    const { id: userId } = req.userData
+    const userStocks = await stocks.fetchUserStocks(userId)
+    return res.json({
+      message: 'User stocks returned successfully!',
+      userStocks
+    })
```

Comment:
Can we have the special message for the case when there `userStocks` is `[]` something like `No stocks found`? (Using ternary operator)

Reply:
Is it really needed? I mean in all other places as well we are just doing this same thing.

```javascript
return res.json({
message: '<something> returned successfully!',
<whatever you are requesting (doesnt matter if its [] or [{...data}]>
})
```

Reply:
If I have 0 zero stocks in my account, wouldn't the message `User stocks returned successfully!` confuse me? It says that it has returned stocks successfully but the stocks array is `[]`

Reply:
6310a35